tests/test_dse.py

```diff
@@ -1411,7 +1411,7 @@ def test_fullopt_w_mpi(self):
 
     @switchconfig(profiling='advanced')
     @pytest.mark.parametrize('space_order,expected', [
-        (8, 173), (16, 307)
+        (8, 173), (16, 309)
```

Comment:
how are they different now?

fun that it only happens with the higher order one

Reply:
Probably due to different order, the `evalf` was on the full fancy equation and may have been doing slughtly different stuff.